app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module MailerHelper
+  def mail_if_subscribed(recipient_email, subject, user)
```

Comment:
Can we use [before_action](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-callbacks) as most of our email need to have this check ?
```suggestion
  def mail_if_subscribed(recipient_email, subject, user)
```

Reply:
Yup, we use this for 3 mailer classes, I was earlier thinking of defining a method in each block or using this helper method. Will update in next commit 👍

Reply:
@tachyons looks like before_action approach wont work here, unable to get user through params nor can we pass params in before action .

If we use a private method to check this condition then again we will require another conditional inside the mailer methods.

So I think the former approach, before this helper method one is better. I have updated approach like it was done before.